tools/c7n_azure/c7n_azure/resources/cost_management_export.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2019 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_azure.actions.base import AzureBaseAction
+from c7n_azure.provider import resources
+from c7n_azure.query import QueryResourceManager
+from c7n_azure.utils import ThreadHelper
+
+from c7n.filters import Filter
+from c7n.utils import get_annotation_prefix as gap
+from c7n.utils import type_schema
+import logging
+
+import datetime
+
+
+log = logging.getLogger('c7n.azure.cost-management-export')
+
+
+@resources.register('cost-management-export')
+class CostManagementExport(QueryResourceManager):

```

Comment:
Are we able to tag these resources or do we need to set `enable_tag_operations` to `False`?

Reply:
This is not an ARM resource, `enable_tag_operations` is required only for Arm resources (we don't add tag actions to non-arm)